bugbug/bug_features.py

```diff
@@ -224,6 +224,11 @@ def __call__(self, bug, **kwargs):
         return sum(commit['files_modified_num'] for commit in bug['commits'])
 
 
+class commit_author_experience(object):
+    def __call__(self, bug, **kwargs):
+        return sum(commit['author_experience'] for commit in bug['commits'])
```

Comment:
Actually, here we can't sum. If you have two patches from the same author, you don't want the sum of the same experience.
We probably want the average here.

Reply:
Right, fixed that.